Header header
string sender_id
string holder_id
